ProjectSettings/GraphicsSettings.asset

```diff
@@ -39,10 +39,10 @@ GraphicsSettings:
   m_SpritesDefaultMaterial: {fileID: 10754, guid: 0000000000000000f000000000000000,
     type: 0}
   m_ShaderSettings_Tier1:
-    useCascadedShadowMaps: 1
-    standardShaderQuality: 2
-    useReflectionProbeBoxProjection: 1
-    useReflectionProbeBlending: 1
+    useCascadedShadowMaps: 0
+    standardShaderQuality: 0
+    useReflectionProbeBoxProjection: 0
+    useReflectionProbeBlending: 0
   m_ShaderSettings_Tier2:
     useCascadedShadowMaps: 1
     standardShaderQuality: 2
```

Comment:
Wonder how these got changed. We should use the version you've got, as they're the correct settings.